client/src/Questions/QuestionsList.js

```diff
@@ -0,0 +1,48 @@
+import { useState, useEffect } from "react";
+import Api from "../../Api";
+import { Link } from "react-router-dom";
+const QuestionsList = () => {
+  const [questions, setQuestions] = useState([]);
+
+  useEffect(() => {
+    Api.questions.index().then((response) => setQuestions(response.data));
+  }, []);
+
+  const onDelete = (questions) => {
+    if (window.confirm(`Are you sure you want to delete ${questions.topic}?`)) {
+      // delete button from the API
+      Api.questions.delete(questions.id).then(() => {
+        // We are fukterunf gthe section list, keeping every section that does not match the one we're deleting
+        const newQuestions = questions.filter((s) => s.id !== question.id);
+        setQuestions(newQuestions);
+      });
+    }
+  };
+  return (
+    <main>
+      <h2>The Quesitons List</h2>
+      <ul>
+        {questions.map((questions) => (
```

Comment:
Typo, the variable in the parentheses should be question SINGULAR, as it represents ONE question inside the questions (plural) array.